dev/build/windows/makecoq_mingw.sh

```diff
@@ -1386,7 +1386,7 @@ function make_coq {
   make_ocaml
   make_num
   make_findlib
-  make_camlp5
+  # make_camlp5
```

Comment:
I am not quite sure this is correct with respect to the building of addons (e.g. doesn't elpi still rely on camlp5?). Since the Windows build is currently broken apparently, we won't know for sure. Thus, I suggest to delay the changes to this file in a subsequent PR (e.g. the Dune + Windows one).

Reply:
As far as I can see this is correct, the CI passed in the past, and elpi doesn't seem to be a supported addon.

Anyways we really need to do something about the mess `dev/build/windows` has become, starting by removing the huge amount of cruft living there.

I will indeed further additions of addons until we refactor the addon building code there; it will likely involve expressing the dependencies better (via a `CI_addon_DEPS` variable?) Then the script should be able to call the build procedure.

Reply:
I confirm the windows failure is "spurious" to this PR, it is mtac2 that is failing but I cannot arrive to understand the error, for some reason `Unicoq` is not found but it should be.